src/deploy/table_report.sql

```diff
@@ -0,0 +1,24 @@
+-- Deploy maevsi:table_report to pg
+
+BEGIN;
+
+CREATE TABLE maevsi.report (
+    id              UUID PRIMARY KEY NOT NULL DEFAULT gen_random_uuid(),
+    reporter_id     UUID NOT NULL REFERENCES maevsi.account(id),
+    reason          TEXT NOT NULL,
+    event_id        UUID REFERENCES maevsi.event(id),
+    upload_id       UUID REFERENCES maevsi.upload(id),
+    user_id         UUID REFERENCES maevsi.account(id),
+    CHECK (num_nonnulls(event_id, upload_id, user_id) = 1),
+    UNIQUE (reporter_id, event_id, upload_id, user_id)
+);
+
+COMMENT ON TABLE maevsi.report IS 'A report.';
+COMMENT ON COLUMN maevsi.report.id IS 'The report''s internal id.';
```

Comment:
```suggestion
COMMENT ON COLUMN maevsi.report.id IS E'@omit create,update\nThe report''s internal id.';
```
This way Postgraphile does not offer to set the UUID so that it's generated server side only.